plugins/source/aws/resources/services/iam/roles.go

```diff
@@ -82,30 +77,6 @@ func getRole(ctx context.Context, meta schema.ClientMeta, resource *schema.Resou
 	return nil
 }
 
-func resolveIamRolePolicies(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
-	r := resource.Item.(*types.Role)
-	cl := meta.(*client.Client)
-	svc := cl.Services().Iam
-	input := iam.ListAttachedRolePoliciesInput{
-		RoleName: r.RoleName,
-	}
-	policies := map[string]*string{}
-	paginator := iam.NewListAttachedRolePoliciesPaginator(svc, &input)
-	for paginator.HasMorePages() {
-		response, err := paginator.NextPage(ctx)
-		if err != nil {
-			if cl.IsNotFoundError(err) {
-				return nil
-			}
-			return err
-		}
-		for _, p := range response.AttachedPolicies {
-			policies[*p.PolicyArn] = p.PolicyName
-		}
-	}
-	return resource.Set("policies", policies)
-}
```

Comment:
This is moved to `plugins/source/aws/resources/services/iam/role_attached_policies.go`